### PR TITLE
Tracking block images

### PR DESCRIPTION
**Pull request by vauxhall**
Added new tracking events

### CODE REVIEW

File: Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift [hunk 1]
```diff
@@ -109,9 +110,11 @@ extension PhotonActionSheetProtocol {
         let noImageMode = PhotonActionSheetItem(title: imageModeTitle, iconString: iconString, isEnabled: noImageEnabled) { action,_ in
             NoImageModeHelper.toggle(isEnabled: action.isEnabled, profile: self.profile, tabManager: self.tabManager)
             if noImageEnabled {
-                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesDisabled)
+                Analytics.shared.browser(.disable, label: .blockImages)
+//                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesDisabled)
```
Comment (jman82) [inline Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift:114]: Shall we remove this line?

File: Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift [hunk 2]
```diff
@@ -109,9 +110,11 @@ extension PhotonActionSheetProtocol {
...
             } else {
-                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesEnabled)
+                Analytics.shared.browser(.enable, label: .blockImages)
+//                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesEnabled)
```
Comment (jman82) [inline Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift:117]: And this too? Maybe if we want to keep it for future generations we can add a comment :hugs:

**Review by jman82 — APPROVED** [review]
Tracking skillz!!!! :rocket:

File: Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift (hunk 1 above)
Comment (vauxhall) [reply Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift:114]: This is part of the Firefox and their Telemetry framework. Commenting out is usually the way we remove them.

File: Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift (hunk 2 above)
Comment (vauxhall) [reply Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift:117]: Same here, commenting might make easier future merges with upstream.

File: Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift (hunk 1 above)
Comment (vauxhall) [reply Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift:114]: Commenting this out is just for clarity, we deactivated the framework, so even if it is not commented it should not perform any action. But having it commented makes it clearer that we are not tracking other things.

**Pull request merged**